toolkits/global/packages/global-expander/__tests__/expander.spec.js

```diff
@@ -19,7 +19,7 @@ const createKeydownEvent = key => {
 			event.key = 'Enter';
 			break;
 		case 'Space':
-			event.key = 'Space';
+			event.key = ' ';
```

Comment:
_googles_ 😆 
Nice catch. Have you checked this works in IE? I read a few things saying it needs 'Spacebar' 🤷‍♂️

Reply:
😱 

Of course IE would do something different 😩

Reply:
added 'Spacebar' to get it detected by older browsers that use this